zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -275,7 +293,14 @@ impl<'a> PrecomputedTxData<'a> {
         };
         let tx_data: zp_tx::TransactionData<PrecomputedAuth> = alt_tx
             .into_data()
-            .map_authorization(f_transparent, IdentityMap, IdentityMap);
+            // FIXME: do we need to pass another arg values or orchard_zsa and issue instead of IdentityMap?
+            .map_authorization(
+                f_transparent,
+                IdentityMap,
+                IdentityMap,
+                IdentityMap,
+                IdentityMap,
+            );
```

Comment:
This seems fine, the transparent outputs are needed for the `TransparentAuthorizingContext` impl used via the `zp_tx::sighash::signature_hash()` call just below this impl block, where the shielded bundles aren't accessed at all for v5 transactions and only the `SaplingAuth` type is constrained for v4.